src/lib/utils/jsonResponses/Forms.ts

```diff
@@ -0,0 +1,50 @@
+//https://docs.synapse.org/rest/org/sagebionetworks/repo/model/form/FormGroup.html
+export type FormGroup = {
+  groupId?: string
+  name: string
+  createdBy?: string
+  createdOn?: string
+}
+
+// see https://docs.synapse.org/rest/org/sagebionetworks/repo/model/form/FormData.html
+export type FormData = {
+  formDataId?: string
+  etag?: string
+  groupId: string
+  name: string
+  createdBy?: string
+  createdOn?: string
+  modifiedOn?: string
+  dataFileHandleId:string
+  submissionStatus: SubmissionStatus
+}
+
+// see https://docs.synapse.org/rest/org/sagebionetworks/repo/model/form/SubmissionStatus.html
+export type SubmissionStatus = {
+  submittedOn: string,
+  reviewedOn: string,
+  reviewedBy: string,
+  state: StatusEnum,
+  rejectionMessage: string
+}
+
+// see https://docs.synapse.org/rest/org/sagebionetworks/repo/model/form/StatusEnum.html
+export enum StatusEnum {
+  WAITING_FOR_SUBMISSION = 'WAITING_FOR_SUBMISSION',
+  SUBMITTED_WAITING_FOR_REVIEW = 'SUBMITTED_WAITING_FOR_REVIEW',
+  ACCEPTED = 'ACCEPTED', 
+  REJECTED = 'REJECTED'
+}
+
+// see https://docs.synapse.org/rest/org/sagebionetworks/repo/model/form/ListRequest.html
+export type ListRequest = {
+  filterByState?: StatusEnum[],
+  groupId: string,
+  nextPageToken?: string
+}
+
+// see https://docs.synapse.org/rest/org/sagebionetworks/repo/model/form/ListResponse.html
+export type ListResponse = {
```

Comment:
this could be made generic and then typed in the actual usage since thats how the API is constructed

Reply:
could you elaborate on this point?

Reply:
Sorry for not being clear, I meant that this could be re-written as 

```
export type ListResponse<T> = {
   page: T [],
   nextPageToken?: string
}
```
and then on this [line](https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/228/files#diff-870197c271aeaea05bb0cef7bc9a65e7R11) and the next you could write them as 

```
 myFormDataList<FormData>
 formAdminFormDataList<FormData>
```

Reply:
But ListResponse is org.sagebionetworks.repo.model.**form**.ListResponse, where the page data object type must be FormData.  
If we made it configurable, the developer who uses this method (Alina in this case) would need to set the response type, and what if they specify a type other than FormData?  It would compile, but fail at runtime (because the response object would not be the expected type).

Reply:
whoops! I was wrong, earlier I though I was looking at a docs page that said page<Entity> with a type param, but clearly not... please ignore my comment.